OpenRA.Mods.Common/Traits/PaletteEffects/FlashPaletteEffect.cs

```diff
@@ -0,0 +1,61 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Drawing;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Used for stormy weather effects.")]
+	class FlashPaletteEffectInfo : ITraitInfo
+	{
+		public readonly string[] ExcludePalettes = { "cursor", "chrome", "colorpicker", "fog", "shroud" };
+
+		public readonly float LightningChance = 0.99f;
```

Comment:
I agree with this.
A lower chance value naturally implies the even will (probably) occur less often.